src/frontend/src/app/components/login/login.component.css

```diff
@@ -0,0 +1,33 @@
+main {
+  max-width: 480px;
+  margin: auto;
+  background-color: #00256133;
```

Comment:
Is it a correct color? HEX color usually has 3 or 6 digits, but here are 8

Reply:
Exactly, the '33' in the end is the alpha (transparency) value